src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_FindReferences_Current.cs

```diff
@@ -28,47 +28,42 @@ public static partial class SymbolFinder
         {
             using (Logger.LogBlock(FunctionId.FindReference, cancellationToken))
             {
-                // If ProjectId is null then this is a call through our old public API.  We don't have
-                // the necessary data to effectively run the call out of proc.
-                if (symbolAndProjectId.ProjectId != null)
```

Comment:
this is effectively a whitespace change below.  However, the meat is this line.  We no longer require that we are given a project in order to do do the FAR op.